site/content/deployment/openshift.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "OpenShift"
+weight = 310
```

Comment:
`weight` should be in the 500s range as defined by the weight of the `Deployments` entry in the menu.